include/swift/SILOptimizer/Analysis/ARCAnalysis.h

```diff
@@ -230,7 +230,7 @@ class ConsumedArgToEpilogueReleaseMatcher {
     Optional<ArrayRef<SILInstruction *>> getFullyPostDomReleases() const {
       if (releases.empty() || foundSomeButNotAllReleases())
         return None;
-      return {releases};
+      return ArrayRef<SILInstruction *>{releases};
```

Comment:
`llvm::makeArrayRef` would be a better choice, since it avoids repeating the element type.

Reply:
I'm not sure if the type deduction would get that right.  `releases` already is an `ArrayRef`, it just is of a derived type and this is changing the type.  In fact, there is another instance of it about 5 lines below this.

Reply:
Ah, I see. Okay, thanks.